src/Test/L0/ProcessExtensionL0.cs

```diff
@@ -49,30 +49,37 @@ public async Task SuccessReadProcessEnv()
 
                     try
                     {
-                        await Task.Delay(2000);
                         trace.Info($"Read env from {timeout.Id}");
-                        var value = timeout.GetEnvironmentVariable(hc, envName);
-                        if (string.Equals(value, envValue, StringComparison.OrdinalIgnoreCase))
+                        int retries = 5;
+                        while (retries > 0)
                         {
-                            trace.Info($"Find the env.");
-                            return;
-                        }
-                        else
-                        {
-                            Assert.True(false, "Expected environment '" + envValue + "' did not match actual '" + value + "'");
+                            try
+                            {
+                                var value = timeout.GetEnvironmentVariable(hc, envName);
+                                Assert.True(string.Equals(value, envValue, StringComparison.OrdinalIgnoreCase), "Expected environment '" + envValue + "' did not match actual '" + value + "'");
+                                break;
+                            }
+                            catch (Exception ex)
+                            {
```

Comment:
It might be helpful for debugging purposes to print the Exception.

Reply:
we do down below in the subsequent catch.